src/amclient/am-client.ts

```diff
@@ -48,8 +48,8 @@ export class AmClient {
     return Axios
       .get(`${this.serverAddress}/json/realms/root/agents/${agentId}`, {
         headers: {
-          host: this.hostname,
-          cookie: `${cookieName}=${sessionId}`
+          Host: this.hostname,
+          Cookie: `${cookieName}=${sessionId}`
```

Comment:
HTTP headers are case in-sensitive.

Reply:
yeah, I just want to make it consistent with other headers' names which were having first letter capital.